File: python/svix/models/sink_status.py
# this file is @generated
from enum import Enum


class SinkStatus(str, Enum):
    ENABLED = "enabled"
    PAUSED = "paused"
    DISABLED = "disabled"
    RETRYING = "retrying"

    def __str__(self) -> str:
        return str(self.value)

File: csharp/Svix/Models/SinkStatus.cs
// this file is @generated
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Svix.Models
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum SinkStatus
    {
        [EnumMember(Value = "enabled")]
        Enabled = 1,

        [EnumMember(Value = "paused")]
        Paused = 2,

        [EnumMember(Value = "disabled")]
        Disabled = 3,

        [EnumMember(Value = "retrying")]
        Retrying = 4,
    }
}
